### ISSUE TITLE
[BUDI-6844] CLI Fails On Docker Compose Plugin

### ISSUE THREAD

**Issue by shawnweeks** [Budibase/budibase#10222]
**Checklist**
- [X] I have searched budibase discussions and github issues to check if my issue already exists

**Hosting**
N/A

**Describe the bug**
Running `budi hosting --init` fails on a system that only has the new Docker Compose Plugin. Docker Compose V1 aka the `docker-compose` command is EOL June 2023

```
[cloud-user@budibase budibase]$ budi hosting --init
Error - docker/docker-compose has not been installed, please follow instructions at: https://docs.budibase.com/docs/docker-compose
```

**To Reproduce**
1.  Install Docker CE on Rocky 8 from https://docs.docker.com/engine/install/centos/
2. Install NodeJS via `sudo dnf module install nodejs:18`
3. Install Budi CLI via `sudo npm install -g @budibase/cli`
4. Try and run `budi hosting --init`

**Expected behavior**
Installation should work normally

**Screenshots**
N/A
  
**App Export**
N/A

**Desktop (please complete the following information):**
N/A

**Additional context**
N/A

<sub>[BUDI-6844](https://linear.app/budibase/issue/BUDI-6844/cli-fails-on-docker-compose-plugin)</sub>

**Comment by mike12345567** [comment]
Hi @shawnweeks - you can track the implementation of support for the docker compose plugin on this issue: https://github.com/PDMLab/docker-compose/pull/228

We utilise this package for our docker-compose integration in the CLI, once it is updated we will be able to support this.

**Comment by shawnweeks** [comment]
When dealing with a issue on a current release of something I don't typically look through all the closed issues on GitHub. Probably should leave at least one of these issues open since Docker is actively telling you not to install docker-compose anymore.

**Comment by AlexZeitler** [comment]
Just wanted to let you know we just released `v0.24.0` of `docker-compose` with `docker compose` plugin support.
Please let us know if you find any issues.

**Comment by mike12345567** [comment]
@AlexZeitler - thanks for letting us know - we'll get updated!
